effects/shaders/chromakey.frag

```diff
@@ -15,17 +15,24 @@ uniform int mode;
  
 float rgb2y (vec3 c) { 
 	/*a utility function to convert colors in RGB into YCbCr*/ 
-	return (0.299*c.r + 0.587*c.g + 0.114*c.b);
+
+// This isn’t colour managed and is a huge mess, but in the
+// short term, using correct weights will give significantly
+// more ideal results. The correct weights for REC.709 are
+// 0.2126 R, 0.7152 G, and 0.722 B. Easy change.
+// The coefficients for YCbCr are calculated off of the
+// REC.709 values.
+	return (0.2126*c.r + 0.7152*c.g + 0.0.722*c.b);

```

Comment:
Simple typo. `0.0.722*c.b` should be `0.722*c.b` (otherwise the effect would not work)

Reply:
Whoops, should be `0.0722`.

Reply:
```suggestion
	return (0.2126*c.r + 0.7152*c.g + 0.0722*c.b);
```